.vsts-ci.yml

```diff
@@ -40,20 +40,19 @@ phases:
     ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
       enableMicrobuild: true
     steps:
-    - script: eng\common\cibuild.cmd
+    - script: build.cmd -ci
         -configuration $(_BuildConfig)
         -prepareMachine
       name: Build
       displayName: Build
       condition: succeeded()
-    - ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
-      - task: PublishBuildArtifacts@1
-        displayName: Publish Windows package assets
-        inputs:
-          pathToPublish: $(Build.SourcesDirectory)/artifacts/packages/$(_BuildConfig)/Shipping
-          artifactName: BuiltPackageOutputs
-          artifactType: container
-        condition: eq(variables['_BuildConfig'], 'Release')
+    - task: PublishBuildArtifacts@1
+      displayName: Publish Windows managed assets
+      inputs:
+        pathToPublish: $(Build.SourcesDirectory)/artifacts
+        artifactName: artifacts
```

Comment:
should you give this a more specific name?

Reply:
I wanted it to be the same as the directory, since then I would be able to just unpack it in the same location for my packages build and it would have the same folder structure as the original build. Basically by doing this and then unpacking it on $(Build.SourcesDirectory), I will automatically get a folder called artifacts under my source root so I won't have to rename for my package build to find the right assets.

Reply:
Actually, this should only publish anything under artifacts/bin because you don't need obj, tmp and other directories that get created under artifacts. That will make the artifacts container be fatter and also slow down the build. Then, when you download the assets you can tell the download task to download it into `$(Build.SourcesDirectory)/artifacts/`

Reply:
It seems OK to preserve / restore to the same structure, but I would hope that you could do so without naming the containers the same as the folder, since the folder name alone isn't very descriptive.

Reply:
The only way to preserve directory structure and renaming the container to something else, is to download the artifacts to another intermediate location like Build.ArtifactsStagingDirectory and then adding an extra step to copy the files over to /bin or whatever destination directory.

Reply:
yeah that's what I thought. @ericstj do you still think that its worth adding the extra step for this? or given that reason is it ok to keep the artifacts name as the destination folder?